awscli/examples/rum/get-app-monitor.rst

```diff
@@ -0,0 +1,46 @@
+**To retrieve the complete configuration information for one app monitor**
+
+The following ``get-app-monitor`` retrieves the complete configuration information for one app monitor. ::
+
+	aws rum get-app-monitor \
```

Comment:
Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general